### DISCUSSION TITLE
Multiminer algos

### DISCUSSION THREAD

**Discussion by smyers63** [RainbowMiner/RainbowMiner#2323] [Q&A] [answered]
In RBM, is Multiminer only set up for Argon variants?  That's all I see at the moment but I thought other algos were there not long ago.

**Comment by RainbowMiner** [comment]
In fact, yes - this version of the Multiminer's main strength is the support of all the Argon algorithm variants.

**Comment by smyers63** [comment]
Weren't other algos set up in RBM not long ago?

**Comment by RainbowMiner** [comment, marked as answer]
No, not really. I think you might mix it up with another miner.
Here is the history: https://github.com/RainbowMiner/RainbowMiner/commits/master/Miners/Multiminer.ps1
Added 2019 with only argon2d4096, then, over time, I added more argon algorithms.

**Comment by username-rand** [comment]
In order to do mine multi algos, there needs to be a way to set OC by device by algo combination and Rainbow miner would OC each GPU by a corresponding profile. Each device has OC for each algo combination.  I don’t see a way to do that. Am I missing something?  Most of my rigs have different GPUs.  

For a example profile15-RTX360ti
Algorithm dynexsolve+kheavyhash

Thanks.

**Reply by RainbowMiner** [reply]
> In order to do mine multi algos, there needs to be a way to set OC by device by algo combination and Rainbow miner would OC each GPU by a corresponding profile. Each device has OC for each algo combination. I don’t see a way to do that. Am I missing something? Most of my rigs have different GPUs.
> 
> For a example profile15-RTX360ti Algorithm dynexsolve+kheavyhash
> 
> Thanks.

You can set the profiles in miners.config.txt 

```
    {
      "MainAlgorithm": "DynexSolve",
      "SecondaryAlgorithm": "kHeavyHash",
      "Params": "",
      "MSIAprofile": "",
      "OCprofile": "Profile15",
      "Difficulty": "",
      "Penalty": "",
      "HashAdjust": "",
      "Hash2Adjust": "",
      "Disable": "0",
      "Tuning": "0",
      "ShareCheck": "",
      "Intensity": ""
    },
```

Unfortunately you will have to set it per miner - fortunately there are only a few miners that handle dual mining.

**Reply by username-rand** [reply]
Thanks, that may allow me to do what I want if I set a different profile for each dual algorithm and device.  Thanks.
